### ISSUE TITLE
Failure to download IANA database

### ISSUE THREAD

**Issue by essenciary** [JuliaTime/TimeZones.jl#153]
I consistently get this error on Ubuntu -- and some have reported it to me on Windows. 
```julia
  Building TimeZones → `~/.julia/packages/TimeZones/wytr8/deps/build.log`
┌ Error: Error building `TimeZones`: 
│ [ Info: Downloading 2018e tzdata
│ ERROR: LoadError: IOError: could not spawn `curl -g -L -f -o /home/parallels/.julia/packages/TimeZones/wytr8/deps/tzarchive/tzdata2018e.tar.gz https://www.iana.org/time-zones/repository/releases/tzdata2018e.tar.gz`: no such file or directory (ENOENT)
│ Stacktrace:
│  [1] _jl_spawn(::String, ::Array{String,1}, ::Cmd, ::Tuple{RawFD,RawFD,RawFD}) at ./process.jl:367
│  [2] (::getfield(Base, Symbol("##495#496")){Cmd})(::Tuple{RawFD,RawFD,RawFD}) at ./process.jl:509
│  [3] setup_stdio(::getfield(Base, Symbol("##495#496")){Cmd}, ::Tuple{RawFD,RawFD,RawFD}) at ./process.jl:490
│  [4] #_spawn#494(::Nothing, ::Function, ::Cmd, ::Tuple{RawFD,RawFD,RawFD}) at ./process.jl:508
│  [5] _spawn at ./process.jl:504 [inlined]
│  [6] #run#505(::Bool, ::Function, ::Cmd) at ./process.jl:652
│  [7] run at ./process.jl:651 [inlined]
│  [8] download(::String, ::String) at ./download.jl:48
│  [9] tzdata_download(::String, ::String) at /home/parallels/.julia/packages/TimeZones/wytr8/src/tzdata/download.jl:90
│  [10] macro expansion at ./logging.jl:310 [inlined]
│  [11] #build#32(::Bool, ::Function, ::String, ::Array{String,1}, ::String, ::String, ::String) at /home/parallels/.julia/packages/TimeZones/wytr8/src/tzdata/build.jl:28
│  [12] #build at ./none:0 [inlined]
│  [13] build(::String, ::Array{String,1}) at /home/parallels/.julia/packages/TimeZones/wytr8/src/tzdata/build.jl:72
│  [14] #build#5(::Bool, ::Function, ::String, ::Array{String,1}) at /home/parallels/.julia/packages/TimeZones/wytr8/src/TimeZones.jl:116
│  [15] build at /home/parallels/.julia/packages/TimeZones/wytr8/src/TimeZones.jl:116 [inlined] (repeats 2 times)
│  [16] top-level scope at none:0
│  [17] include at ./boot.jl:317 [inlined]
│  [18] include_relative(::Module, ::String) at ./loading.jl:1038
│  [19] include(::Module, ::String) at ./sysimg.jl:29
│  [20] include(::String) at ./client.jl:388
│  [21] top-level scope at none:0
│ in expression starting at /home/parallels/.julia/packages/TimeZones/wytr8/deps/build.jl:6
└ @ Pkg.Operations /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:1068
```

**Comment by essenciary** [comment]
The problem is that `curl` is not installed by default: 
```
parallels@parallels-vm:~$ curl -g -L -f -o /home/parallels/.julia/packages/TimeZones/wytr8/deps/tzarchive/tzdata2018e.tar.gz https://www.iana.org/time-zones/repository/releases/tzdata2018e.tar.gz
The program 'curl' is currently not installed. You can install it by typing:
sudo apt install curl
```

**Comment by omus** [comment]
I'm using the Julia [`Base.download`](https://github.com/JuliaLang/julia/blob/3c8119f4c1fe3a83a488e04caa834f26432317a2/base/download.jl#L25) and not using `curl` directly. Do you have `wget` or `fetch` installed on your system?

**Comment by essenciary** [comment]
Yes, `wget` comes bundled with the Ubuntu distro.

**Comment by omus** [comment]
Fixed by https://github.com/JuliaLang/julia/commit/d6e43e25ff7ca57ff85342243fd072e51ae303d5 which will be included in [Julia 1.0.1](https://github.com/JuliaLang/julia/pull/28764)
